pkg/controller/controlplane/valuesprovider.go

```diff
@@ -1318,3 +1318,13 @@ func hasDurosStorageNetwork(infrastructure *apismetal.InfrastructureConfig, nws
 	}
 	return false, nil
 }
+
+// GetControlPlaneShootCRDsChartValues returns the values for the control plane shoot CRDs chart applied by the generic actuator.
+// Currently the provider extension does not specify a control plane shoot CRDs chart. That's why we simply return empty values.
+func (vp *valuesProvider) GetControlPlaneShootCRDsChartValues(
```

Comment:
Actually pretty interesting because we have maybe have some CRDs to deploy through this new mechanism.

I only found the azure provider from Gardener that uses this functionality: https://github.com/gardener/gardener-extension-provider-azure/blob/54a217bb96e30fd0eaa8d45a8a94e048c0bbe7e4/pkg/controller/controlplane/valuesprovider.go#L463

Reply:
They also apply volumesnapshots if k8s version is < 1.21, we do the same, but with a different approach